java/src/jmri/jmrit/logix/Engineer.java

```diff
@@ -401,17 +403,13 @@ protected float modifySpeed(float tSpeed, String sType) {
                 log.error("Unknown speed interpretation {}", _speedMap.getInterpretation());
                 throw new java.lang.IllegalArgumentException("Unknown speed interpretation " + _speedMap.getInterpretation());
         }
-        if (log.isDebugEnabled()) log.debug("modifySpeed: from {}, to {}, speedtype= {} using interpretation {}",
-                tSpeed, throttleSpeed, sType, _speedMap.getInterpretation());
+//        if (log.isDebugEnabled()) log.debug("modifySpeed: from {}, to {}, speedtype= {} using interpretation {}",
```

Comment:
Why is this commented out? If its too noisy during debug mode, but valuable to assist in low-level debugging, change it to ```if (log.isTraceEnabled()) log.trace(...```, so it can be enabled without having to recompile.